### PR TITLE
Compile 1-01-01 table

### PR DESCRIPTION
**Pull request by amilan17**
Compile is a branch of all the issues that are related to table 1-01-01.

### CODE REVIEW

File: tables_en/1-01-01.csv [hunk 1]
```diff
@@ -1,449 +1,590 @@
...
+289,gas phase,HF (hydrofluoric acid),"IUPAC: fluorane, PubChem CID: 14917, CAS Number: 7664-39-3, in gas phase"
+291,gas phase,ROOH,
+294,total deposition,Acidity-Alkalinity,
+295,total deposition,Electric conductivity,
+296,total deposition,Hydrogen ion (H+) or pH,
+302,total deposition,S (sulphur),"IUPAC: sulfur, PubChem CID: n/a, CAS Number: 7704-34-9, in total deposition"
+304,wind,Direction of cloud movement,
+305,wind,Gust Speed,
+307,wind,"Wind (Z component, vertical)",Z component of wind vector  (horizontal and vertical profile)
+310,wind,"Upper wind (X, Y components, horizontal)",Horizontal component of wind as a vector in an orthogonal coordinate system.
```
Comment (amilan17) [inline tables_en/1-01-01.csv:86]: #267 - 309 deleted

**Comment by amilan17** [comment]
branch not ready for pr

**Review by fstuerzl — CHANGES_REQUESTED** [review]
Apart from the following issues, the diff looks good. Only these three variables need to be fixed:
- 315, description: Spectral volumetric extinction cross-section of aerosol particles.
- 192, name: CH4 (methane)
- 506, description: Effective depth (thickness) of a cloud from the viewpoint of radiation propagation. Specifically, OD = exp(-K.Δz), where K is the extinction coefficient [km-1] and Δz the vertical path [km] between the base and the top of the cloud. The WMO Cloud Atlas (https://cloudatlas.wmo.int/en/optical-thickness.html) defines cloud optical thickness as 'The optical thickness of a cloud is the degree to which the cloud prevents light from passing through it.'

@amilan17, can I make the corrections myself by editing this branch https://github.com/wmo-im/wmds/blob/compile/tables_en/1-01-01.csv or how does it work?

**Comment by amilan17** [comment]
> @amilan17, can I make the corrections myself by editing this branch https://github.com/wmo-im/wmds/blob/compile/tables_en/1-01-01.csv or how does it work?

@fstuerzl -- please do make the corrections in the compile branch! Thanks.

**Comment by fstuerzl** [comment]
> > @amilan17, can I make the corrections myself by editing this branch https://github.com/wmo-im/wmds/blob/compile/tables_en/1-01-01.csv or how does it work?
> 
> @fstuerzl -- please do make the corrections in the compile branch! Thanks.

Done!

**Comment by fstuerzl** [comment]
Name of variable 619 fixed: "Sulphate (SO4=), corrected"

**Review by joergklausen — APPROVED** [review]
I have reviewed this to the best of my abilities and it looks okay now.

**Review by fstuerzl — APPROVED** [review]
I have reviewed the table again and didn't find any mistakes.

**Pull request merged**